content/en/serverless/distributed_tracing/_index.md

```diff
@@ -2,48 +2,123 @@
 title: Distributed Tracing
 kind: documentation
 further_reading:
-    - link: 'tracing/serverless_functions'
-      tag: 'Documentation'
-      text: 'Installing AWS X-Ray'
+- link: "/tracing/"
+  tag: "Documentation"
+  text: "Explore Datadog APM"
+- link: "/tracing/trace_search_and_analytics/#live-search-for-15-minutes"
+  tag: "Documentation"
+  text: "Live Search"
 ---
 
 {{< img src="tracing/serverless_functions/ServerlessDistributedTrace.png" alt="Trace Serverless Functions"  style="width:100%;">}}
 
-By connecting your serverless traces to metrics, Datadog provides a context-rich picture of your application’s performance, allowing you to better troubleshoot performance issues given the distributed nature of serverless applications.
+By connecting your serverless traces to metrics, Datadog provides a context-rich picture of your application’s performance, allowing you to better troubleshoot performance issues given the distributed nature of serverless applications. 
+
+The Datadog Python, Node.js, Ruby, Go and Java tracing libraries support distributed tracing for AWS Lambda. The easiest way to add tracing to your application is with the [Datadog Lambda Library][2], which includes the Datadog tracing library as a dependency.
```

Comment:
I'm going to add a note here for customers who are looking to get started quickly to go straight to the installation steps